src/index.ts

```diff
@@ -74,7 +75,17 @@ if(firstArg == "validate"){
         guessParameters
     };
 
-    let result = validator.validateFile(secondArg, options);
+    let result = Object();
+    try {
+      result = validator.validateFile(secondArg, options);
+    } catch(err) {
+      console.log('Unable to parse template!');
```

Comment:
I think something to spur the user on to resolve the issue would be helpful here.

"Unable to parse template! Is the template formatted correctly?"
"Unable to parse template! Does the path exist?"
"Unable to parse template! Use --verbose for more information."

Probably the last one is most useful.

Additionally, as we are now catching the file not found, it might be nice to return this as a separate error, so this parsing error is the last case "I've run into terrible difficulty" message, as opposed to something that could be encountered due to a typo.

Reply:
Sounds very nice. I believe that from a UX quality perspective the most specific messages are best for users however, they tend to become tedious to maintain therefore I'd like to lean on a combination of 3 and 2 implying that perhaps we should provide sufficiently specific messages for common scenarios such as invalid path and at the same time provide a broad catch-all type message for not really expected exceptions :)  ... then again it's just a thought I think you have more experience with this project and perhaps see things more clearer than I do.

Reply:
I think

`Unable to parse template! Use --verbose for more information.` would work best for a general fall-through then.

Then we just need to add some extra logic to check if a path exists and return an error nicely.